resources/views/users/index.blade.php

```diff
@@ -90,7 +152,9 @@ class="table table-striped snipe-table"
 
 @stop
 
-@section('moar_scripts')
+@section('moar_scripts')val
```

Comment:
is this `val` supposed to be there?

Reply:
Ack, no, thank you - good catch. My computer is suuuuuper slow right now, and I've got that phpstorm laaaaag.